docs/user-guide.md

```diff
@@ -389,6 +464,10 @@ In this section, you can choose the language model for communication, as well as
 
 ### Language Models
 
+1. In the **Talk to** area, click **See full list...** to display all the available models, applications and assistants.
+2. Select the **Models** tab to display the available language models.
```

Comment:
A little bit confusing...as Models, Assistants and Applications are selected by default. And if to click on the tab, it will be hidden on Talk to.